### ISSUE TITLE
Something can't fit on the page and there is a horizontal scrolling

### ISSUE THREAD

**Issue by lipis** [LeaVerou/bliss#6]
But I'm too lazy to figure out what's wrong.. :)

<img width="1102" alt="screen shot 2015-12-05 at 15 37 02" src="https://cloud.githubusercontent.com/assets/125676/11608313/20daf6f2-9b66-11e5-8ba0-8b1a42265ab5.png">

**Comment by LeaVerou** [comment]
Huh, odd. I can’t reproduce it. Which browser?

**Comment by lipis** [comment]
Yes.. sorry..Chrome.. but I'm on dev channel so you can ignore I guess: Version 48.0.2564.22 dev.

![screen shot 2015-12-05 at 21 10 53](https://cloud.githubusercontent.com/assets/125676/11609950/c1bed060-9b94-11e5-9bd8-cf378bdfb204.png)

**Comment by LeaVerou** [comment]
Can you run `console.log(innerWidth, innerHeight)` in the console and let me know what it says?

**Comment by seebaermichi** [comment]
I recognized the same issue in Firefox 42 on Mac Yosemite and on Windows 10 latest Edge and Firefox. It obviously depends on the browser-window size. 
What I found out so far, it might have something to do with the code within pre- and code-boxes. So in Firefox there is much more space for indentation as in Chrome and it tries to fit code within the box.
![screenshot_blissfuljs_firefox42_mac](https://cloud.githubusercontent.com/assets/2083999/11612311/adc32a10-9bf4-11e5-83e0-b96f68a35cc5.jpg)

While debugging I found another issue in Chrome on Windows 10 and 7. On some code-boxes it shows also horizontal scrollbars.
![screenshot_blissfuljs_chrome_windows7](https://cloud.githubusercontent.com/assets/2083999/11612314/2814cd0a-9bf5-11e5-80a3-15f58257f789.jpg)

I guess the reason is the same, code doesn't fit into the boxes and code has overflow: auto.

Tried to fix it, but I couldn't found a solution without creating other issues.
